tests/fixtures/transformations.py

```diff
@@ -64,15 +65,14 @@ class TransformationTestCase:
     input_data: DataPack
     transformed_model: SchemaPack
     transformed_data: DataPack
+    annotation: BaseModel = field(default_factory=lambda: EMPTY_SUBMISSION_ANNOTATION)
```

Comment:
Could just pass `EmptySubmissionAnnotation` (without the parentheses at the end) instead, so you don't need the lambda